code/modules/mob/living/simple_animal/hostile/retaliate/king_of_goats.dm

```diff
@@ -0,0 +1,328 @@
+	melee_damage_lower = 40
+	melee_damage_upper = 60
+	environment_smash = ENVIRONMENT_SMASH_RWALLS
+	pixel_y = 5
+
+	var/spellscast = 0
+	var/phase3 = FALSE
+	var/sound_id = "goat"
+	var/special_attacks = 0
+	var/list/rangers = list()
+	var/current_song = 'sound/ambience/Visager-Battle.ogg'
+	var/current_song_length = 1200
+	stun_chance = 7
+
+/mob/living/simple_animal/hostile/retaliate/goat/king/phase2/Initialize()
+	. = ..()
+	update_icon()
+
+/mob/living/simple_animal/hostile/retaliate/goat/king/Found(atom/A)
+	if(isliving(A))
+		return A
+	return ..()
+
+/mob/living/simple_animal/hostile/retaliate/goat/guard/Found(atom/A)
+	if(isliving(A))
+		return A
+	return ..()
+
+/mob/living/simple_animal/hostile/retaliate/goat/guard
+	name = "honour guard"
+	desc = "A very handsome and noble beast."
+	icon = 'icons/mob/king_of_goats.dmi'
+	icon_state = "goat_guard"
+	icon_living = "goat_guard"
+	icon_dead = "goat_guard_dead"
+	faction = list("goat_king")
+	attack_same = FALSE
+	sentience_type = SENTIENCE_BOSS
+	stat_attack = DEAD
+	wander = FALSE
+	robust_searching = TRUE
+	health = 125
+	maxHealth = 125
+	minbodytemp = 0
+	maxbodytemp = INFINITY
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	armour_penetration = 10
+	melee_damage_lower = 10
+	melee_damage_upper = 15
+
+/mob/living/simple_animal/hostile/retaliate/goat/guard/master
+	name = "master of the guard"
+	desc = "A very handsome and noble beast - the most trusted of all the king's men."
+	icon_state = "goat_guard_m"
+	icon_living = "goat_guard_m"
+	icon_dead = "goat_guard_m_dead"
+	health = 200
+	maxHealth = 200
+	armour_penetration = 15
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	move_to_delay = 3
+
+/mob/living/simple_animal/hostile/retaliate/goat/guard/pope
+	name = "Goat Pope"
+	desc = "For what is a God without a pope to spread their holy words"
+	icon_state = "goat_pope"
+	icon_living = "goat_pope"
+	icon_dead = "goat_pope_dead"
+	health = 100
+	maxHealth = 100
+	armour_penetration = 25
+	melee_damage_lower = 25
+	melee_damage_upper = 30
+	move_to_delay = 3
+	loot = list(/obj/item/clothing/head/goatpope)
+
+/mob/living/simple_animal/hostile/retaliate/goat/king/Retaliate()
+	..()
+	if(stat == CONSCIOUS && prob(5))
+		visible_message("<span class='warning'>\The [src] bellows indignantly, with a judgemental gleam in his eye.</span>")
+
+/mob/living/simple_animal/hostile/retaliate/goat/king/phase2/Retaliate()
+	set waitfor = FALSE
+	..()
+	if(spellscast < 5)
+		if(prob(5) && move_to_delay >= 3) //speed buff
+			spellscast++
+			visible_message("<span class='cult'>\The [src] shimmers and seems to phase in and out of reality itself!</span>")
+			move_to_delay = 1
+
+		else if(prob(5) && melee_damage_lower != 50) //damage buff
+			spellscast++
+			visible_message("<span class='cult'>\The [src]' horns grow larger and more menacing!</span>")
+			melee_damage_lower = 50
```

Comment:
this doesn't ever reset; is that intentional?

why not just roll this in with the burn damage buff?